### PR TITLE
Added try/catch while removing reciever

### PR DESCRIPTION
**Pull request by Javasamurai**

### CODE REVIEW

File: TeaLeaf/src/com/tealeaf/TeaLeaf.java [hunk 1]
```diff
@@ -507,7 +507,12 @@ public void onWindowFocusChanged(boolean hasFocus) {
 			/* Fix Crash in Android 7.0 Samsung devices where screenOffReceiver is null
 			TODO: Need to find why it is null */
 			if (screenOffReceiver != null) {
-				unregisterReceiver(screenOffReceiver);
+				try {
+					unregisterReceiver(screenOffReceiver);
+				} catch (Exception e) {
+					logger.log("{focus} Unable to unregister screenOffReciever");
+					e.printStackTrace();
```
Comment (rampr) [inline TeaLeaf/src/com/tealeaf/TeaLeaf.java:514]: Why are we doing this ?

Comment (Javasamurai) [reply TeaLeaf/src/com/tealeaf/TeaLeaf.java:514]: There are cases where reciever is already been unregistered,In that case crash was happening.It's safe to put try/catch here.

Comment (Javasamurai) [reply TeaLeaf/src/com/tealeaf/TeaLeaf.java:514]: https://stackoverflow.com/a/9950673/2911650

Comment (vivekb91) [reply TeaLeaf/src/com/tealeaf/TeaLeaf.java:514]: I think try catch is not needed here. But won't cause any harm.

Comment (Javasamurai) [reply TeaLeaf/src/com/tealeaf/TeaLeaf.java:514]: We can remove it as we are already checking for null value.

**Comment by rampr** [comment]
Closing this in favour of #52

**Pull request closed without merging**